packages/next/build/index.ts

```diff
@@ -416,6 +418,9 @@ export default async function build(
                 FONT_MANIFEST
               )
             : null,
+          config.experimental.optimizeCss
+            ? path.join(CLIENT_STATIC_FILES_PATH, CLIENT_STATIC_CSS_PATH, '*')
```

Comment:
This needs to be the actual files since globs aren't currently handled here

Reply:
Thanks. I've fixed this.